.github/workflows/ci.yml

```diff
@@ -9,7 +9,9 @@ env:
 
 jobs:
   quality_checks:
-    uses: ./.github/workflows/quality_checks.yml
+    uses: NHSDigital/eps-workflow-quality-checks/.github/workflows/quality-checks.yml@v2
```

Comment:
Could you change these to `@main`, and remove the `with: node_version` field? That'll be no longer necessary

Reply:
The version has been changed to `main`